package.json

```diff
@@ -9,6 +9,7 @@
     "big.js": "^6.1.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^6.0.2",
```

Comment:
- [ ]  It would be better if you add react-redux to the dependencies, react-redux will be needed to use Redux in the next projects
😄 🔧

Reply:
Thanks I forgot to add it 😄